tests/dashbio_demos/dash-pileup/app.py

```diff
@@ -0,0 +1,200 @@
+                                            genomewideline_value=-np.log10(0.05),
+                                            p='padj',
+                                            snp='SNP',
+                                            gene='Gene',
+                                )
+                            )
+                        ])
+                    ),
+                    dcc.Tab(
+                        label='About',
+                        value='what-is',
+                        children=html.Div(className='control-tab', children=[
+                            html.H4(className='what-is', children='What is pileup.js?'),
+                            dcc.Markdown(
+                                """
+                                The Dash pileup.js component is a high-performance genomics
+                                data visualization component developed originally by the Hammer Lab
+                                (https://github.com/hammerlab/pileup.js). pileup.js
+                                supports visualization of genomic file formats, such as vcfs,
+                                bam, and bigbed files. pileup.js additionally allows flexible interaction
+                                with non-standard data formats. Users can visualize GA4GH JSON formatted
+                                alignments, features and variants. Users can also connect with and visualize
+                                data stored in GA4GH formatted data stores.
+                                """
+                            )
+                        ])
+                    )
+                ]
+            )
+        ]),
+        dcc.Loading(className='dashbio-loading', id='pileup-output', children =
+        html.Div([
+            dash_bio.Pileup(
+                id=_COMPONENT_ID,
+                range=HOSTED_CASE_DICT['rna-differential']['range'],
+                reference = REFERENCE,
+                tracks= HOSTED_CASE_DICT['rna-differential']['tracks'],
+            )
+        ])),
+    ])
+
+
+
+def callbacks(_app):
+
+    @_app.callback(
+    Output(_COMPONENT_ID, 'range'),
+    Input('pileup-dashbio-volcanoplot', 'clickData'))
+    def update_range(point):
+
+        data = HOSTED_CASE_DICT['rna-differential']
+
+        if (point == None):
+            range = data['range']
+        else:
+
+            # get genomic location of selected genes and goto
+            pointText = point['points'][0]['text']
+            gene = pointText.split('GENE: ')[-1]
+
+            row = DE_dataframe[DE_dataframe['Gene'] == gene].iloc[0]
+
+            range = {'contig': row['chr'],
+                    'start': row['start'],
+                    'stop': row['end']}
+
+        return range
+
+
+# only declare app/server if the file is being run directly
+if 'DEMO_STANDALONE' not in os.environ:
+    app = run_standalone_app(layout, callbacks, header_colors, __file__)
+    server = app.server
```

Comment:
```suggestion
app = run_standalone_app(layout, callbacks, header_colors, __file__)
server = app.server
```
Same as the `layout_helper` changes above.